.github/workflows/precommit.yml

```diff
@@ -5,10 +5,15 @@ jobs:
   precommit:
     if: github.repository == 'opensearch-project/OpenSearch'
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
```

Comment:
Workflow failure, guess it requires default values.

> Error when evaluating 'continue-on-error' for job 'precommit'. .github/workflows/precommit.yml (Line: 8, Col: 24): Unexpected value ''

Reply:
oops, that needs to be down in the steps... fixed.